Lib/fontbakery/checkrunner.py

```diff
@@ -816,29 +816,19 @@ def _check_log_override(overrides, status, message):
   result_message = message
   override = False
   for override_target, new_status, new_message_string in overrides:
-    if (
-            # Override by matching Status, bad style but possible (should we???).
-            # Some checks do not define Messages with a code.
-            isinstance(override_target, Status) and result_status == override_target
-          ) \
-          or \
-          (
-            # Override by matching message.code, good style.
-            hasattr(result_message, 'code') and result_message.code == override_target
-          ):
-      override = True
-      # Break the for loop, we had a successful override.
-      break
-
-  if override:
+    # Override is only possible by matching message.code
+    if not hasattr(result_message, 'code')\
+                            or result_message.code != override_target:
+      continue
+    override = True
```

Comment:
shouldn't this result in an error message instead of silently refusing to override?

Reply:
It's just not a match in my eyes. Every not matching override rule is ignored, because it doesn't apply. Should it raise if it would be a matching status? Why to put a status in there in the first place, we don't support or document it anywhere any more. Also, `override_target` could be anything, should it only raise if it is a status type or also if it is, a say a number type or just not a string type (or the wrong string type)?